Realm/RLMAccessor.mm

```diff
@@ -549,40 +549,40 @@ static void RLMSuperSet(RLMObjectBase *obj, NSString *propName, id val) {
     superSetter(obj, prop.setterSel, val);
 }
 
-// getter/setter for unmanaged
-static IMP RLMAccessorStandaloneGetter(RLMProperty *prop, RLMAccessorCode accessorCode) {
+// getter/setter for standalone
```

Comment:
I agree. I didn't want to touch the actual code so close to release. We can go back later and rename everything properly.